doc/cli/npm-docs.md

```diff
@@ -8,6 +8,8 @@ npm-docs(1) -- Docs for a package in a web browser maybe
     npm home [<pkgname> [<pkgname> ...]]
     npm home .
 
+    aliases: home
```

Comment:
Out of scope for this PR, but `npm docs` and `npm home` are semi-different conceptually, so we may want to document them separately at some point. Also, for this PR, it might make sense to nuke the two `npm home` examples, in much the same way that you did with `npm tst` below.